docs/source/pipelines.mdx

```diff
@@ -12,7 +12,7 @@ specific language governing permissions and limitations under the License.
 
 # Optimum pipelines for inference
 
-The [`~pipelines.pipeline`] function makes it simple to use models from the [Model Hub](https://huggingface.co/models) for accelerated inference on a variety of tasks such as text classification.
+The [`~pipelines.pipeline`] function makes it simple to use models from the [Model Hub](https://huggingface.co/models) for accelerated inference on a variety of tasks such as question answering or image classification.
```

Comment:
```suggestion
The [`~pipelines.pipeline`] function makes it simple to use models from the [Model Hub](https://huggingface.co/models) for accelerated inference on a variety of tasks such as text classification, question answering, and image classification.
```
I think that it makes sense to list them all, as they are typical and quite different examples.

Reply:
Actually all the supported tasks are listed a few lines below. My point was to show that it is not only NLP and not only classification (which is why I replaced "text classification" by "question answering"). But your suggestion is fine, I'm going to integrate it :)